service/de_encoding/ciphering.go

```diff
@@ -19,22 +23,38 @@ func (en AESEncoder) Decrypt(ct string) string {
 
 	c, err := aes.NewCipher(en.key)
```

Comment:
Just a reminder that after our discussion we agreed on passing `key` with each method encrypt/decrypt invocation since an encoder could be reused by different users with different passwords.

Reply:
resolved. now passed with each method